.travis.yml

```diff
@@ -0,0 +1,31 @@
+sudo: false
+dist: trusty
+language: erlang
+otp_release:
+  - 20.0.1
+  - 19.3.6.1
+  - 18.3
+addons:
+  apt:
```

Comment:
[Travis WebLint](http://lint.travis-ci.org/) reports: “in addons section: unexpected key apt, dropping”. Probably because of that it says also: “value for addons section is empty, dropping”.

Reply:
Probably because this is beta. Please see https://travis-ci.org/lucafavatella/gb_merkle_trees/builds/271726232

Reply:
Ok, it’s a false warning then.